README.md

```diff
@@ -1263,6 +1263,29 @@ address families to Lua.
 * `"NETDEV"`: Device ingress and egress path
 * `"BRIDGE"`: Ethernet Bridge.
 
+### `completion`
+
+The `completion` library provides functions to handle task completion in Lua.
```

Comment:
Can you add a link to Linux kernel documentation if any or to its implementation otherwise.. we are trying to have these links on every binding.. you can refer to luathread, for instance..